pages/index.html

```diff
@@ -1,12 +1,170 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+<html lang="en" class="bg-night">
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" type="text/css" href="../stylesheets/index-style.css">
+		<link rel="stylesheet" type="text/css" href="../stylesheets/style.css">
+		<title>HomePage</title>
+	</head>
+
+	<body class="no-margin">
+		<div class="container">
+			<nav class="navbar white">
+				<div class="navbar-container">
+					<h1 class="no-margin"><a class="no-style" href="index.html">Hannah's Personal Website</a></h1>
+					<div class="menu-icon">
+						<input id="menu" class="no-margin" type="checkbox">
```

Comment:
🤯 Wow!  Very cool use of the checkbox 'checked' property to hid/reveal the menu!